meson_options.txt

```diff
@@ -302,6 +302,8 @@ option('gshadow', type : 'boolean',
        description : 'support for shadow group')
 option('default-locale', type : 'string', value : '',
        description : 'default locale used when /etc/locale.conf does not exist')
+option('nspawn-locale', type : 'string', value : 'C.UTF-8',
+       description : 'default locale used by systemd-nspawn when executing commands in a container')
```

Comment:
I think it is not necessary to introduce a new knob, but please reuse `default-locale`.

Reply:
This was discussed in an earlier PR where I tried to do this and it was rejected. There's no guarantee that the default locale from the host is also the right locale to use for nspawn. I'm on mobile now but I'll link the PR when I get back to a PC.

Reply:
See discussion here: https://github.com/systemd/systemd/pull/23252

Reply:
OK. Thank you for the pointer.

Reply:
From https://github.com/systemd/systemd/pull/23252#issuecomment-1118678927
> I suspect that if the payload has no support for `C.UTF-8`, it'll fall back to `C` anyway. So this shouldn't matter too much. Let's keep it as is, and if people start complaining because their old payloads emit a warning about unknown locale, we can always make this (separately) configurable.

That's make sense for me.